src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/nonLinearThermalDiffusion_TemperatureDependentSinglePhaseThermalConductivity/Example.rst

```diff
@@ -0,0 +1,62 @@
+.. _AdvancedExampleWellboreNonLinearThermalDiffusionTemperatureDependentSinglePhaseThermalConductivity:
+
+
+#####################################################################################################################
+Non-Linear Thermal Diffusion Around a Wellbore: The Case with Temperature Dependent Single Phase Thermal Conductivity
+#####################################################################################################################
+
+------------------------------------------------------------------
+Problem description
+------------------------------------------------------------------
+
+This example is an extension of the linear thermal diffusion problem presented in :ref:`AdvancedExamplePureThermalDiffusionWellbore`. It uses the thermal single-phase flow solver to model a non-linear thermal diffusion problem around a wellbore where the single phase thermal conductivity of the porous rock depends linearly on the temperature.
+
+
+**Input file**
+
+This benchmark example uses no external input file and everything required is
+contained within two GEOS xml files that are located at:
+
+.. code-block:: console
+
+  inputFiles/singlePhaseFlow/thermalCompressible_2d_base.xml
+
+and
+
+.. code-block:: console
+
+  inputFiles/singlePhaseFlow/thermalCompressible_temperatureDependentSinglePhaseThermalConductivity_benchmark.xml
+
+
+In this example, we focus on the ``Constitutive`` tag.
+
+-----------------------------------------------------------
+Constitutive
+-----------------------------------------------------------
+
+The reference value of the single phase thermal conductivity of the porous medium around the wellbore and its derivative with respect to temperature are defined in the ``SolidInternalEnergy`` XML block:  
+
+.. literalinclude:: ../../../../../../../inputFiles/singlePhaseFlow/thermalCompressible_2d_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_SinglePhaseThermalConductivity_nonLinear -->
+  :end-before: <!-- SPHINX_SinglePhaseThermalConductivity_nonLinearEnd -->
+
+
+---------------------------------
+Results and benchmark
+---------------------------------
+
+A good agreement between the results obtained using GEOS and the analytical results is shown in the figure below:
+
+
+.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/nonLinearThermalDiffusion_TemperatureDependentSinglePhaseThermalConductivity/plot.py
```

Comment:
To avoid unexpected doc build failure, it is better to give a unique name for the python script of each case.

Reply:
Done